### ISSUE TITLE
Seems like k8 skydns is still trying to use etcd

### ISSUE THREAD

**Issue by KaGeN101** [mantl/mantl#1938]
Does not seem like k8 has any dns capability due to the fact that skydns tries to access etcd and not consol

This could be causing a whole host of hidden issues.

```
[centos@mantl-control-01 ~]$ kubectl logs kube-dns-v8-v63w1 kube2sky
I0421 15:50:25.187998       1 kube2sky.go:383] [Attempt: 1] Attempting access to etcd after 5 second sleep
I0421 15:51:00.188494       1 kube2sky.go:383] [Attempt: 2] Attempting access to etcd after 5 second sleep
I0421 15:51:35.189011       1 kube2sky.go:383] [Attempt: 3] Attempting access to etcd after 5 second sleep
I0421 15:52:10.189499       1 kube2sky.go:383] [Attempt: 4] Attempting access to etcd after 5 second sleep
I0421 15:52:45.189961       1 kube2sky.go:383] [Attempt: 5] Attempting access to etcd after 5 second sleep
I0421 15:53:20.190469       1 kube2sky.go:383] [Attempt: 6] Attempting access to etcd after 5 second sleep
```

- Git commit hash or branch: f04d06ded13bfcd6b57600a0ee5e5f28cede5e15
- Cloud Environment: AWS

**Comment by KaGeN101** [comment]
This is not an issue since there is etcd on the control. The problem is nothing on the k8 network can reach this due to the dns key issue.

Pretty sure if that issue is fixed everything in k8 will just work

**Comment by Theaxiom** [comment]
@KaGeN101 Check out the branch feature/k8-ui

**Comment by Theaxiom** [comment]
We will have to do a little lifting to switch back the keystore from Consul to etcd, I believe the first part will be using a more recent hyperkube image.

**Comment by KaGeN101** [comment]
This is not an issue as it is only for its own existence. it just could not access the etcd on control nodes. Fixed in master #1963. Closing the issue.
